moped-editor/src/views/projects/projectView/ProjectComponents/DeleteComponentModal.js

```diff
@@ -32,15 +32,13 @@ const useStyles = makeStyles((theme) => ({
 const DeleteComponentModal = ({
   showDialog,
   clickedComponent,
-  setClickedComponent,
   setIsDeletingComponent,
   onDeleteComponent,
 }) => {
   const classes = useStyles();
 
   const onClose = () => {
     setIsDeletingComponent(false);
-    setClickedComponent(null);
```

Comment:
This prevents the click component item in the list from collapsing when closing the delete modal. All the other modals work this way.

Reply:
i like this, thanks for making it consistent

Reply:
ditto!